pkg/collector/corechecks/system/disk/iostats_pdh_windows_test.go

```diff
@@ -68,7 +68,7 @@ func TestIoCheckWindows(t *testing.T) {
 	addDefaultQueryReturnValues()
 
 	ioCheck := new(IOCheck)
-	ioCheck.Configure(nil, nil, "test")
+	ioCheck.Configure(1, nil, nil, "test")
```

Comment:
nit (that can apply to all the test that you have modified)

instead of using `1`, you can create a `const fakeConfigHash = 1` to make it more clear what is this `1`